ote/src/cljs/ote/ui/tabs.cljs

```diff
@@ -11,17 +11,20 @@
   Assumes that first tab is selected by default."
 
   [tabs {:keys [update-fn selected-tab] :as actions}]
-  [:div.tab {:style {:padding-bottom "20px"}}
-   [:ul {:style {:list-style "none" :padding-bottom "12px"}}
+  [:div.tab {:style {:padding-bottom "40px"}}
     (doall
       (for [{:keys [label value]} tabs]
         ^{:key (str label)}
-        [:li {:style    {:display "inline"}
-              :on-click #(update-fn value)}
-         [:span (if (or (= value selected-tab) (and (nil? selected-tab) (= value (:value (first tabs)))))
+        [:div ;(merge
```

Comment:
Remove leftover comments ` ; ` and discards  `#_` in this section.

Reply:
yep. my bad